static/js/redux/ui/top_bar_advising.jsx

```diff
@@ -78,6 +79,28 @@ class TopBarAdvising extends React.Component {
   }
 
   render() {
+    const ReturnToScheduleButton = (
+      <div className="cal-btn-wrapper">
+        <a href="/">
+          <button
+            className="import-data"
+            data-tip
+            data-for="return-to-schedule-btn-tooltip"
+          >
+            <p style={{ color: 'gray' }}>Return&nbsp;to&nbsp;Schedule</p>
+          </button>
+        </a>
+        <ReactTooltip
+          id="return-to-schedule-btn-tooltip"
+          class="tooltip"
+          type="dark"
+          place="bottom"
+          effect="solid"
+        >
+          <span>Return to Schedule</span>
+        </ReactTooltip>
+      </div>
+    );
     return (
       <div className="top-bar">
         <a href="/" className="semesterly-name">
```

Comment:
I think it is this - we don't want this anymore as a link since we have the "Return to Schedule" button

Reply:
Here is what it looks like after removing the href:
![image](https://user-images.githubusercontent.com/37627810/115048689-d1b59f00-9ea7-11eb-9be3-48e517f5e94c.png)